content/docs/glossary/muxed-accounts.mdx

```diff
@@ -0,0 +1,305 @@
+
+There are many other benefits to embedding this abstraction into the protocol:
+
+  - **"Share-ability"**: Rather than worrying about error-prone things like copy-pasting memo IDs, you can just share your `M...` address.
+  - **SDK support**: The various [SDKs](../software-and-sdks) support this abstraction natively, letting you create, manage, and work with muxed accounts easily.
+  - **Efficiency**: By combining related virtual accounts under a single account's umbrella, you can avoid holding reserves and paying fees for all of them to exist in the ledger individually.
+
+<Alert>
+
+  **Terminology Tip**: The term *muxing* comes from computer networking and telecommunications, where multiple signals are combined into one signal over a shared medium. We are doing the same thing here with muxed accounts: combining multiple "virtual" accounts into a single shared account in the Stellar ledger.
+
+</Alert>
+
+It's worth noting that this feature is intended to be a high-level abstraction, merely embedded into the protocol for convenience and standardization. There's no validation on IDs: as far as the Stellar Network is concerned, all of the [supported operations][supported-ops] operate exactly as if you did not used a muxed account. For example, if you make a payment from two muxed accounts that share an underlying Stellar account (but have different IDs), this is *exactly the same* as that Stellar account sending two payments as far as the ledger is concerned.
+
+It's worth noting that this feature is intended to be a high-level abstraction, embedded into the protocol for convenience and standardization. As far as the Stellar Network is concerned, all of the [supported operations][supported-ops] operate exactly as if you did *not* use a muxed account. For example, if you make a payment from two "sibling" muxed accounts (i.e. muxed accounts sharing an underlying `G...` account but with different IDs), this is *exactly the same* as that Stellar account sending two payments, as far as the ledger is concerned.
+
+
+## Supporting Opt-In
+
+Because muxed account support is still experimental, you need to **explicitly opt-in** to creating operations with muxed properties. If you do not pass these opt-in flags in the necessary places, SDKs will throw errors. This is done in an effort to ensure that muxed accounts are not "accidentally" working in your application and thus may be misleading to your users.
+
+Each SDK implements this a little differently, but generally-speaking you need to explicitly call out muxing support when creating operations and/or transactions that use muxed accounts.
```

Comment:
It's set for every op/tx, yeah. Should I add in "each" in this last sentence to drive that home?

> generally-speaking you need to explicitly call out muxing support when creating each operation and/or transaction that uses muxed accounts.

Or do you think it needs more clarity?

Reply:
> Should I add in "each" in this last sentence to drive that home?
I think that's all that's needed. 

Side question: will the flag remain in perpetuity or is this a temporary setting?

Reply:
I don't know the answer to that, honestly. If I had to surmise, it'll be a two-phase process: first, `withMuxing: true` will be the default (i.e. opt-**out** instead of opt-in). Then, we'll be able to drop the flag entirely.